src/Analyzers/Core/Analyzers/Helpers/HashCodeAnalyzer/HashCodeAnalyzer.OperationDeconstructor.cs

```diff
@@ -61,15 +48,15 @@ public bool TryAddHashedSymbol(IOperation value, bool seenHash)
                         //
                         // This is generated by both VS and ReSharper.  Though each use different mathematical
                         // ops to combine the values.
-                        return _hashCodeVariable != null &&
+                        return hashCodeVariable != null &&
```

Comment:
📝 these renames severely detract from the desirability of this feature. I would prefer we not move in this direction.

Reply:
They seem entirely readable to me :-).  We went 10+ years with this what be the naming for fields. Only changing because of the .net naming rules.  But those rules apply to fields. These are parameters, and are named correctly based on that :-)

Reply:
I'm not a big fan either. The convention of using `_` prefix for instance fields and `s_` for static is helpful in making it clear that one refers to state of the object vs. a parameter of the current method. It's true that we also used non-prefixed identifiers for protected fields. I did not like that. 

Unless the type declaration is small enough to fit on one screen, it will be harder to see whether or not the identifier refers to the object state or a parameter.

Reply:
I'm struggling to think of a case where I would care :-)